opentelemetry-exporter-geneva/geneva-uploader-ffi/Cargo.toml

```
@@ -5,7 +5,23 @@ edition = "2021"
 license = "Apache-2.0"
 rust-version = "1.75.0"
 
+[lib]
+crate-type = ["cdylib", "staticlib", "rlib"]
+
 [dependencies]
+geneva-uploader = { path = "../geneva-uploader" }
+opentelemetry-proto = { workspace = true, default-features = false, features = ["logs", "gen-tonic-messages"] }
+tokio = { version = "1.0", features = ["rt", "rt-multi-thread", "sync", "time", "net"] }
```

I don't these features are needed.

```suggestion
tokio = { version = "1.0", features = ["rt-multi-thread"] }
```